catalog/neon-prod-earthengine/projects_neon-prod-earthengine_assets_HSI_REFL_001_original.txt

```diff
@@ -0,0 +1,3855 @@
+local id = 'projects/neon-prod-earthengine/assets/HSI_REFL/001';
```

Comment:
Could you remove this file?

Reply:
I thought I did this, in [61121d6](https://github.com/google/earthengine-catalog/pull/1111/commits/61121d654953d021cc02f0dcd45d1e3e893debca). Checking.